packages/manager-react-components/src/components/filters/filter-add.component.tsx

```diff
@@ -41,15 +48,22 @@ export function FilterAdd({ columns, onAddFilter }: Readonly<FilterAddProps>) {
       {
         key: selectedId,
         comparator: selectedComparator,
-        value,
+        value:
+          selectedColumn?.type === FilterTypeCategories.Date
+            ? dateValue?.toISOString()
+            : value,
+        type: selectedColumn?.type,
```

Comment:
Is it possible that selectedColumn and dateValue could be undefined ? If no there is no need for `?`

Reply:
That should not happen